contracts/wrappers/examples/Counter.ts

```diff
@@ -164,6 +166,10 @@ export class ContractClient implements Contract, typeAndVersion.Interface {
     })
   }
 
+  static async code(): Promise<Cell> {
```

Comment:
imo bindings should be decoupled from the env and the backend they are binding to. The same bindings might work against N versions of the same contract if the public API doesn't change (or across different contracts when binding a trait like Ownable) - so which version is the binding actually compiling?